CIP-0086/README.md

```diff
@@ -217,11 +215,8 @@ by the new explicit assignment.
 
 ### Simple metadata updates
 
-A simple metadata update transaction must only send ADA
-from the oracle update address to itself
-and must not mint any tokens,
-but it may contain any number of inputs and outputs.
-Otherwise, the transaction is ignored for the purposes of token metadata.
+A simple metadata update transaction may contain any inputs and outputs,
+as these are ignored for the purposes of updating token metadata.
```

Comment:
```suggestion
A simple metadata update transaction must be signed
by the signing key of the oracle update address,
but it may contain any inputs and outputs,
as these are ignored for the purposes of updating token metadata.
```